### PR TITLE
Add a "Getting Started" section to the README

### PR DESCRIPTION
**Pull request by gurnben**
## Summary of Changes

This PR adds a simple, brief "Getting Started" to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,10 +39,56 @@ make plugin
...
+2. Run `cm create cph --api-url=<api> --console=<console-url> --group=<rbac-group> --namespace=<namespace-containing-clusters> <name-of-cph>` and run `cm create cph --help` to view all options
+3. Run `cm get cph` to verify that your active clusterpoolhost is correct, `cm set cph <clusterpoolhost-name>` to swap clusterpoolhosts, and `cm use cph <clusterpoolhost-name>` to switch to that clusterpoolhost's context.  
+
+### Working with Clusters
+
+#### Using ClusterPools
+
+ClusterPools maintain a configurable and scalable number of OpenShift clusters in a hibernating state.  `cm` exposes the capability to create, view, consume clusters from, and destroy clusterpools.  
+
+To create a ClusterPool, see `cm create <clusterpool/cp> --help` for all options.  You can veiw your created ClusterPool(s) with `cm get <clusterpool/cp>`.
```
Comment (dhaiducek) [inline README.md:64]:
```suggestion
To create a ClusterPool, see `cm create <clusterpool/cp> --help` for all options.  You can view your created ClusterPools with `cm get <clusterpool/cp>`.
```

File: README.md [hunk 2]
```diff
@@ -39,10 +39,56 @@ make plugin
...
 This CLI (and plugin) is still in development, but aims to expose OCM/ACM's functional through a useful and lightweight CLI and kubectl/oc CLI plugin.  Some features may not be present, fully implemented, and it might be buggy!  
 
+## Getting Started
+
+### Setting up a ClusterPoolHost
+
+In order to work with clusters, you need set up `cm` with your hub cluster(s) - `cm` refers to these hubs as "clusterpoolhost"(s) or "cph"(s) for short!  
+
+To set up your first ClusterPoolHost:
+1. `oc login` to your ClusterPoolHost running [Red Hat Advanced Cluster Management](https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes), [Multicluster Engine for Kubernetes](https://github.com/open-cluster-management/backplane-operator), or [Open Cluster Management](http://github.com/open-cluster-management-io).  **Your user must be able to create ServiceAccounts in the target namespace, given that `create cph` creates a ServiceAccount.  Also ensure that ServiceAccounts in that namespace have the relevant access such as create/delete clusterclaims, pools, etc.
```
Comment (dhaiducek) [inline README.md:54]: The "relevant access" and "etc" is a little bit hand-wavy. Do we have a `hive` link we could point to for that?

```suggestion
1. `oc login` to your ClusterPoolHost running [Red Hat Advanced Cluster Management](https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes), [Multicluster Engine for Kubernetes](https://github.com/open-cluster-management/backplane-operator), or [Open Cluster Management](http://github.com/open-cluster-management-io).  **Your user must be able to create ServiceAccounts in the target namespace, given that `create cph` creates a ServiceAccount.  Also ensure that ServiceAccounts in that namespace have the relevant access such as create/delete ClusterClaims, ClusterPools, etc.   
```

File: README.md [hunk 3]
```diff
@@ -39,10 +39,56 @@ make plugin
...
+2. Run `cm create cph --api-url=<api> --console=<console-url> --group=<rbac-group> --namespace=<namespace-containing-clusters> <name-of-cph>` and run `cm create cph --help` to view all options
+3. Run `cm get cph` to verify that your active clusterpoolhost is correct, `cm set cph <clusterpoolhost-name>` to swap clusterpoolhosts, and `cm use cph <clusterpoolhost-name>` to switch to that clusterpoolhost's context.  
+
+### Working with Clusters
+
+#### Using ClusterPools
+
+ClusterPools maintain a configurable and scalable number of OpenShift clusters in a hibernating state.  `cm` exposes the capability to create, view, consume clusters from, and destroy clusterpools.
```
Comment (dhaiducek) [inline README.md:62]:
```suggestion
ClusterPools maintain a configurable and scalable number of OpenShift clusters in a hibernating state.  `cm` exposes the capability to create, view, consume clusters from, and destroy ClusterPools.  
```

File: README.md [hunk 4]
```diff
@@ -39,10 +39,56 @@ make plugin
...
+
+### Working with Clusters
+
+#### Using ClusterPools
+
+ClusterPools maintain a configurable and scalable number of OpenShift clusters in a hibernating state.  `cm` exposes the capability to create, view, consume clusters from, and destroy clusterpools.  
+
+To create a ClusterPool, see `cm create <clusterpool/cp> --help` for all options.  You can veiw your created ClusterPool(s) with `cm get <clusterpool/cp>`.
+
+Once you have a clusterpool, you can claim clusters from the pool for use using `cm create <clusterclaim/cc> <clusterpool-name> <clusterclaim-name>`, see `cm create <clusterclaim/cc> --help` for more options.  You can also view ClusterClaims and details with `cm get <clusterclaim/cc>` and delete ClusterClaims with `cm delete <clusterclaim/cc>`.
```
Comment (dhaiducek) [inline README.md:66]:
```suggestion
Once you have a ClusterPool, you can claim clusters from the pool for use using `cm create <clusterclaim/cc> <clusterpool-name> <clusterclaim-name>`, see `cm create <clusterclaim/cc> --help` for more options.  You can also view ClusterClaims and details with `cm get <clusterclaim/cc>` and delete ClusterClaims with `cm delete <clusterclaim/cc>`.  
```

File: README.md [hunk 5]
```diff
@@ -39,10 +39,56 @@ make plugin
...
+
+To set up your first ClusterPoolHost:
+1. `oc login` to your ClusterPoolHost running [Red Hat Advanced Cluster Management](https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes), [Multicluster Engine for Kubernetes](https://github.com/open-cluster-management/backplane-operator), or [Open Cluster Management](http://github.com/open-cluster-management-io).  **Your user must be able to create ServiceAccounts in the target namespace, given that `create cph` creates a ServiceAccount.  Also ensure that ServiceAccounts in that namespace have the relevant access such as create/delete clusterclaims, pools, etc.   
+2. Run `cm create cph --api-url=<api> --console=<console-url> --group=<rbac-group> --namespace=<namespace-containing-clusters> <name-of-cph>` and run `cm create cph --help` to view all options
+3. Run `cm get cph` to verify that your active clusterpoolhost is correct, `cm set cph <clusterpoolhost-name>` to swap clusterpoolhosts, and `cm use cph <clusterpoolhost-name>` to switch to that clusterpoolhost's context.
```
Comment (dhaiducek) [inline README.md:56]: Add `delete` command here also.

File: README.md [hunk 6]
```diff
@@ -39,10 +39,56 @@ make plugin
...
+
+To create a ClusterPool, see `cm create <clusterpool/cp> --help` for all options.  You can veiw your created ClusterPool(s) with `cm get <clusterpool/cp>`.
+
+Once you have a clusterpool, you can claim clusters from the pool for use using `cm create <clusterclaim/cc> <clusterpool-name> <clusterclaim-name>`, see `cm create <clusterclaim/cc> --help` for more options.  You can also view ClusterClaims and details with `cm get <clusterclaim/cc>` and delete ClusterClaims with `cm delete <clusterclaim/cc>`.  
+
+Finally, you can delete a ClusterPool with `cm delete <clusterpool/cp>`.  
+
+#### Creating Clusters
+
+To create individual clusters with specific configurations, you can use `cm create cluster`, see `cm create cluster --help` for more options, including a `values.yaml` template.
```
Comment (dhaiducek) [inline README.md:72]: Does `create cluster` require a ClusterPool? Is this different from `create clusterclaim`? If this is distinct, I think it needs a clearer title and/or be moved to the bottom of this section just before "Bringing clusters under management" and away from the sections describing ClusterPools.

File: README.md [hunk 7]
```diff
@@ -39,10 +39,56 @@ make plugin
...
+
+Once you have a clusterpool, you can claim clusters from the pool for use using `cm create <clusterclaim/cc> <clusterpool-name> <clusterclaim-name>`, see `cm create <clusterclaim/cc> --help` for more options.  You can also view ClusterClaims and details with `cm get <clusterclaim/cc>` and delete ClusterClaims with `cm delete <clusterclaim/cc>`.  
+
+Finally, you can delete a ClusterPool with `cm delete <clusterpool/cp>`.  
+
+#### Creating Clusters
+
+To create individual clusters with specific configurations, you can use `cm create cluster`, see `cm create cluster --help` for more options, including a `values.yaml` template.  
+
+#### Nagivating with Clusters
```
Comment (dhaiducek) [inline README.md:74]: Should this title be "ClusterClaims" instead of "Clusters"

File: README.md (hunk 2 above)
Comment (KevinFCormier) [inline README.md:54]: It might be helpful to include the full list of resources that require permission, as well as the API group and version.

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Great additions! I have some comments. 🙂

File: README.md (hunk 7 above)
Comment (KevinFCormier) [inline README.md:74]:
```suggestion
#### Navigating with Clusters
```

**Review by KevinFCormier — COMMENTED** [review]
A few comments, typo corrections

File: README.md (hunk 6 above)
Comment (gurnben) [reply README.md:72]: This actually creates a ClusterDeployment and ManagedCluster resource!  I've updated the title to read "Creating Managed Clusters".

File: README.md (hunk 7 above)
Comment (gurnben) [reply README.md:74]: It _shoud_ also work with ManagedClusters, so I just left it as "clusters".  Plus it works with ClusterPoolHosts.

File: README.md (hunk 2 above)
Comment (gurnben) [reply README.md]: I'll add as much detail as I can but alas its... a bit lossy.

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**